frontend/integration-tests/test-cypress.sh

```diff
@@ -90,10 +90,10 @@ if [ -n "${nightly-}" ] && [ -z "${pkg-}" ]; then
 fi
 
 if [ -n "${headless-}" ] && [ -z "${pkg-}" ]; then
-  yarn run test-cypress-console-headless
-  yarn run test-cypress-dev-console-headless
-  yarn run test-cypress-olm-headless
-  yarn run test-cypress-helm-headless
+  # yarn run test-cypress-console-headless
+  # yarn run test-cypress-dev-console-headless
+  # yarn run test-cypress-olm-headless
+  # yarn run test-cypress-helm-headless
```

Comment:
why are all of these tests disabled?

Reply:
The test were disabled temporarily due to investigate shipwright tests in isolation on CI.